src/include/rbd/features.h

```diff
@@ -11,6 +11,7 @@
 #define RBD_FEATURE_DATA_POOL           (1ULL<<7)
 #define RBD_FEATURE_OPERATIONS          (1ULL<<8)
 #define RBD_FEATURE_MIGRATING           (1ULL<<9)
+#define RBD_FEATURE_NON_PRIMARY         (1ULL<<10)
```

Comment:
@dillaman need to update python's `test_rbd.test_features_to_string`, which supposes 1024 is invalid feature:
```
======================================================================
FAIL: test_rbd.test_features_to_string
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/mgolub/ceph/ceph.ci/src/test/pybind/test_rbd.py", line 134, in _require_new_format
    return fn(*args, **kwargs)
  File "/home/mgolub/ceph/ceph.ci/src/test/pybind/test_rbd.py", line 500, in test_features_to_string
    assert_raises(InvalidArgument, rbd.features_to_string, features)
AssertionError: InvalidArgument not raised by features_to_string
```
And I suppose the updated test should be backported so it would not fail on upgrade tests?

Reply:
Looks like that test is only in the master branch. I bumped it forward 14 bits to give us some room.